### ISSUE TITLE
Improve docs

### ISSUE THREAD

**Issue by BrOrlandi** [KasselLabs/simple-do-spaces#3]
We need to document as JSDoc to clarify methods usage and arguments.

All improvements to docs are welcome.

**Comment by HarisSpahija** [comment]
Hey BrOrlandi, I would like to pick this up! Do you want to have JSDocs on all files?

**Comment by BrOrlandi** [comment]
Yes, so it will be shown on IntelliSense in VS Code. It will be very helpful when using this lib.

Also the docs in README.md can be improved too.
